ccpage108.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Create a glossary of terms in Python.
+# Use a loop to run through the dictionary's keys and values.
+python_glossary = {
+    'list': 'collection of items',
+    'range': 'generates series of numbers',
+    'input': 'get information from user',
+    'len': 'find out umber of characters in an object',
+    'for_loop': 'do a repetitve task',
+    'conditional': 'a decision making point',
+}
+
+for term, definition in python_glossary.items():
+    print(f"\nTerm: {term}")
+    print(f"Definition: {definition}")
```

Comment:
I'd put the `\n` here instead of at the top. Accomplishes the same thing, but conventionally at the end.